### PR TITLE
Changed routing so markdown page is possible as landing page, added O…

### PR DESCRIPTION
**Pull request by KiLanger**
…verview, First Steps and Requirements Page

### CODE REVIEW

File: docs/docs/install-manuals/first-steps.mdx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+Builds the containers using git submodules. This allows versioning and reflects code changes in the respective submodules.
+
+```sh
+docker compose -f docker-compose-dev.yaml up
+```
+
+### Kubernetes
+
+@Leon
```
Comment (lesser1) [inline docs/docs/install-manuals/first-steps.mdx]: Idk if we should commit tags or To-Dos on the main branch. I would suggest we do not but create an issue for what's missing.

File: docs/docs/install-manuals/first-steps.mdx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+This approach does not reflect changes in the source code. It always starts the current (published) version of the services.
+
+```sh
+docker compose up
+```
+
+##### Building containers
+
+Builds the containers using git submodules. This allows versioning and reflects code changes in the respective submodules.
```
Comment (lesser1) [inline docs/docs/install-manuals/first-steps.mdx:32]: Someone please correct me if I'm wrong but I thought one must execute the `git submodule update` command with the arguments `--init` and `--recursive` in order to setup all the submodules first before building the system.

File: docs/docs/install-manuals/requirements.mdx [hunk 3]
```diff
@@ -0,0 +1,17 @@
+---
+id: requirements
+title: System Requirements
+sidebar_position: 1
+---
+
+Since the microservice reference architecture consists of multiple microservices, a performant PC or Server is required to run all services.
+
+### Hardware requirements
```
Comment (lesser1) [inline docs/docs/install-manuals/requirements.mdx:9]: Maybe we should add a note that this is only an estimate and that we cannot guarantee anything. Just to be on the safe side.

**Review by lesser1 — CHANGES_REQUESTED** [review]
Overall, good work! I like it! Especially the overview. Since I'm not an expert regarding web development I'm unsure about the changes to index.js. I just don't know whether or not that's the standard way to solve this "problem" of wanting a specific page to be the landing page. However, the solution you provided seems quite elegant to my eyes. Regarding the "Request Changes" status: It's not that I demand changes, I just think we should discuss what I commented on.

File: docs/docs/install-manuals/first-steps.mdx (hunk 1 above)
Comment (KiLanger) [reply docs/docs/install-manuals/first-steps.mdx]: In the moment, docs is not a component in Gropius but I will add it.

File: docs/docs/install-manuals/first-steps.mdx (hunk 2 above)
Comment (KiLanger) [reply docs/docs/install-manuals/first-steps.mdx:32]: Good catch

File: docs/docs/install-manuals/requirements.mdx (hunk 3 above)
Comment (KiLanger) [reply docs/docs/install-manuals/requirements.mdx:9]: I mean we know that with the current amount of microservices that it runs on the docker setup with 16 GB of RAM. I have checked the docker disk usage, which is around 7GB. I have left out the CPU as I have no idea there.
The only thing for which I also do not know the requirements would be a lokal kubernetes setup.

Comment (lesser1) [reply docs/docs/install-manuals/requirements.mdx:9]: Okay, fine for me then.

**Review by lesser1 — APPROVED** [review]

**Pull request merged**